zerver/openapi/zulip.yaml

```diff
@@ -4533,6 +4533,19 @@ paths:
                                             In Zulip 7.0 (feature level 159), `Nobody` was added as an option to
                                             `edit_topic_policy` enum.
                                         - $ref: "#/components/schemas/GroupSettingValue"
+                                    can_resolve_topics_group:
+                                      allOf:
+                                        - description: |
+                                            A [group-setting value](/api/group-setting-values) defining
+                                            the set of users who have permission to resolve topics
+                                            of a stream.
+
+                                            This setting cannot be set to `"role:internet"` and `"role:owners"`
+                                            system groups.
```

Comment:
This line is not required -- it's both incorrect in this case given the configuration above, but also the "group-setting-values" page documents how to determine based on data the server provides which values are valid (without needing to worry about changes to it appearing only in docs); so we actively don't want lines like this anymore.

Reply:
sure, removed it!